lib/web_ui/lib/src/engine.dart

```diff
@@ -161,6 +162,9 @@ void webOnlyInitializeEngine() {
   // initialize framework bindings.
   domRenderer;
 
+  // Calling this getter to force the [WebExperiments] instance to be created.
+  webExperiments;
```

Comment:
`domRenderer` is actually a bad pattern to mimic. We had several bugs due to unpredictable initialization order. Consider using the `ensureInitialized` pattern from the various "Binding" classes we have.